### PR TITLE
Fix hashchange for IE8+

### PR DESCRIPTION
**Pull request by jdlehman**
Fix rute change for browsers that do not have `oldURL` and `newURL` properties on the `hashchange` event

### CODE REVIEW

**Comment by jdlehman** [comment]
@whatknight This is what I was thinking as a solution. It still needs tests (I through it together quickly before I sign off for the day), but can you see if that works?

**Comment by jdlehman** [comment]
ps :beers: for removing Karma!

File: src/router.js [hunk 1]
```diff
@@ -13,9 +13,14 @@ import {
   isFalse
 } from './utils';
 
+let IEoldURL = '';
+
 function handleRouteChange(e) {
-  const oldPath = pathFromHash(e.oldURL);
-  const newPath = pathFromHash(e.newURL);
+  const oldURL = e.oldURL || IEoldURL;
+  const newURL = e.newURL || window.location.hash;
```
Comment (whatknight) [inline src/router.js]: this was my original approach. the problem with this is that the hash could be changed multiple times before the event is actually handled, causing the new url to be wrong and causing handlers to fire incorrectly.

Comment (jdlehman) [inline src/router.js]: Ah makes sense. I will close this and we can move forward with your PR. :+1:

**Pull request closed without merging**